src/lib/metrics/Metrics.ts

```diff
@@ -181,8 +181,8 @@ export class Metrics {
           errorData: txErrorData,
         },
         env: {
-          VITE_APP_IS_HOME_SITE: import.meta.env.VITE_APP_IS_HOME_SITE ?? null,
-          VITE_APP_VERSION: import.meta.env.VITE_APP_VERSION ?? null,
+          IS_HOME_SITE: IS_HOME_SITE ?? null,
+          VERSION: APP_VERSION ?? null,
```

Comment:
let's use APP_VERSION just in case?
I think there might be some other version types (builder version, git version, etc) that can be confused with just "version"